05 - HTML/README.md

```diff
@@ -1 +1,437 @@
+
+And finally we have the `select` which will contain a `option` for every option you can select from the list:
+
+```
+<select>
+    <option value="1">One</option>
+    <option value="2">Two</option>
+    <option value="3">Three</option>
+    <option value="4">Four</option>
+    <option value="5">Five</option>
+</select>
+```
+
+You see that `value` attribute there? That's the real value that will be sent, the text inside the `option` tag it's only a display value for the user.
+
+To finish with **tags** I will show you some that has no visual impact for website but are very useful:
+
+The `link` tag will help you to import external resources to your site. This will helping us to having more control and a readable code. This usually go inside the `head` tag and goes together with a `src` attribute. The value of this attribute will be pointing to the resource we want to include it.
+
+```
+<link href="resource.path">
+```
+
+Adding comments to the HTML code is useful to explain others what are you doing or probably explaining why you take that approach. To add comments you need the following syntax:
+
+```
+<!-- Comment here -->
+```
+
+And last but not least the `meta` tag. This generally will have a couple of attributes: `name` and `content`. The `meta` tag is usually used several times with different values. Here you can define keywords, description, author, etc. from the your website.
+
+This is the way you do:
+
+```
+<meta name="author" content="Nearsoft Apprentice">
+<meta name="description" content="BEST WEBSITE EVER">
+<meta name="keywords" content="html, tags, html, keywords, blah, blah, blah">
+```
+
+Search engines will look for these meta tags in order to show your website in their results, and also you can use meta tags for specific social networks i.e. Facebook:
+
+```
+<meta property="og:url" content="https://nearsoft.com/home-b/?utm_expid=927768-22.tHy6zLtaTvy4k26gC6OS-A.1/">
+<meta property="og:title" content="Grow your Development Team">
+<meta property="og:site_name" content="Nearsoft">
+<meta property="og:description" content="">
+<meta property="og:type" content="website">
+<meta property="og:image" content="https://nearsoft.com/wp-content/themes/Velvet/_img/nearsoft-symbol.png">
+<meta property="og:locale" content="en_us">
+```
+
+Twitter uses it's own too:
+
+```
+<meta property="twitter:account_id" content="10445872">
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@nearsoft">
+<meta name="twitter:title" content="Grow your Development Team">
+<meta name="twitter:description" content="…">
+<meta name="twitter:image" content="">
+```
+
+As summary you must know now that HTML in the skeleton for our website, and will help you to display the information that you want in several ways, inserting graphics, tables, lists, paragraphs, etc. and also you can define the divisions of the layouts inside your html file.
```

Comment:
`As summary you must know now that HTML in the skeleton`

could be?

`As summary you must know now that HTML is the skeleton`